emission/integrationTests/docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: "3"
+services:
+  web-server:
+    build: 
+      context: .
+    depends_on:
+      - db
+      - nominatim
+    environment:
+      - DB_HOST=db
+      - WEB_SERVER_HOST=0.0.0.0
+      - NOMINATIM_QUERY_URL=http://rhodeisland-nominatim:8080
+      - GFBK_KEY=$GFBK_KEY
+    volumes:
+      # specify the host directory where the source code should live
+      # If this is ~/e-mission-server-docker, then you can edit the files at
+      # ~/e-mission-server-docker/src/e-mission-server/emission/...
+      # - CHANGEME:/src/
+      - ../..:/src/e-mission-server
+    networks:
+       - emission
+       - nominatim
+
+  db:
+    image: mongo:4.4.0
+    ports:
+      - "1234:1234"
+    deploy:
+      replicas: 1
+      restart_policy:
+        condition: on-failure
+
+    #Volumes is the preferred way to persist data generated by a container. In this case we use a volume to persist the contents
+    #of the data base. Learn more about how to use volumes here: https://docs.docker.com/storage/volumes/
+    # And learn how to configure volumes in your compose file here: https://docs.docker.com/compose/compose-file/#volume-configuration-reference
+    volumes:
+      - mongo-data:/data/db
+    networks:
+       - emission
+  #adding section to incorporate nominatim server functionality
+  nominatim:
+    image: nataliejschultz/rhodeisland-image:2.0
+    container_name: rhodeisland-nominatim
+    ports:
+      - "8080:8080"
+      # healthcheck currently commented out, since it isn't necessary and keeps going the entire time the container is running. 
+    healthcheck:
+      test: ["CMD", "curl", "-v", "http://localhost:8080"]
+      interval: 1m
+      timeout: 15s
+      retries: 3
```

Comment:
it looks like the healthcheck is now commented in. Update comment?

Reply:
Commented out now, not necessary with how fast the container is ready!